src/openforms/registrations/contrib/objects_api/tests/test_api_group_model.py

```diff
@@ -6,10 +6,10 @@
 from django.test import TestCase, override_settings
 from django.utils.translation import gettext as _
 
+from openforms.contrib.objects_api.models import ObjectsAPIGroupConfig
```

Comment:
I see all these places that I was importing the `ObjectsAPIGroupConfig` from the registrations module and I am trying to remember why I had done it in this way..maybe beacuse of the migrations problems?

Reply:
yeah I also think it was migrations related, I know it was deliberate!